packages/reputation-miner/ReputationMiner.js

```diff
@@ -254,7 +255,10 @@ class ReputationMiner {
     // TODO: Include updates for all child skills if x.amount is negative
     // We update colonywide sums first (children, parents, skill)
     // Then the user-specifc sums in the order children, parents, skill.
-    await this.insert(colonyAddress, skillId, userAddress, score, updateNumber);
+
+    // Converting to decimal, since its going to be converted to hex inside `insert`
+    const skillIdDecimal = new BN(skillId, 16).toString();
```

Comment:
In case of skill id of 16:
I would get hex value [here](https://github.com/JoinColony/colonyNetwork/pull/294/files/8b9c8fb595fb476066b1ba198c9ad552039831e9#diff-9c228e1a1f7fe6683164300c770b62ffR254) (0000000000000000000000000000000000000000000000000000000000000010)
and then inside `insert` it will be converted to hex again and we will get (000000000000000000000000000000000000000000000000000000000000000a)
which is wrong skill id

Reply:
Oh darn, yeah. I guess this stuff hasn't actually seen such high skill ids before - that's entirely on me. This feels like a bit of a hacky solution, though; I think a more robust solution would be for `breakKeyInToElements` to return `0x`-prefixed strings.